src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -12,6 +12,8 @@ import { getContainerSecretsFolders, getHostSecretsFolders } from '../../debuggi
 import { NetCoreDebugOptions } from '../../debugging/netcore/NetCoreDebugHelper';
 import { vsDbgInstallBasePath } from '../../debugging/netcore/VsDbgHelper';
 import { ext } from '../../extensionVariables';
+import { getContainerOSFromPlatformOS } from "../../runtimes/docker/utils/getContainerOsString";
+import { getNativeArchitecture } from "../../runtimes/docker/utils/getNativeArchitecture";

```

Comment:
should we moved `getContainerOSFromPlatformOS` and `getNativeArchitecture` to src/utils ?

Reply:
or to `TaskHelper.ts?`

Reply:
I think that entirely depends on whether we end up using `normalizeContainerOS` in `withDockerPlatformArg` or not.

Reply:
let's have it stay in runtimes/docker.utils since we're using them in withDockerPlatformArg.ts